shell/common/rasterizer.cc

```diff
@@ -89,6 +89,14 @@ void Rasterizer::Setup(std::unique_ptr<Surface> surface) {
         delegate_.GetTaskRunners().GetRasterTaskRunner()->GetTaskQueueId();
     raster_thread_merger_ =
         fml::MakeRefCounted<fml::RasterThreadMerger>(platform_id, gpu_id);
+
+    raster_thread_merger_->SetMergeUnmergeCallback(
```

Comment:
Does the merge/unmerge callback need to be removed during rasterizer teardown?

The callback is holding a pointer to the rasterizer, so the rasterizer needs to ensure that the callback does not outlive the rasterizer.

Reply:
Done in https://github.com/flutter/engine/pull/20965/commits/2893c3434cc1498bd592154a7fb7def9efe5e90c